website/src/index-page.tsx

```diff
@@ -1,119 +1,397 @@
+              style={{
+                // TODO: default line height is 1, but that cuts of the "g" in the title
+                lineHeight: 1.2,
+              }}
+            >
+              GraphQL Yoga
+            </h1>
+            <p className="max-w-screen-sm mx-auto mt-6 text-2xl text-gray-600 text-center dark:text-gray-400">
+              The fully-featured GraphQL Server with focus on easy setup,
+              performance and great developer experience.
+            </p>
+            <div className="mt-10 flex flex-col sm:flex-row items-center justify-center gap-4">
+              <ButtonLink href="/v3">Documentation</ButtonLink>
+              <ButtonLink className="hidden lg:block" href="/tutorial/basic">
+                Tutorial
+              </ButtonLink>
+              <ButtonLink
+                className="flex flex-row gap-2 items-center"
+                href="https://github.com/dotansimha/graphql-yoga"
+              >
+                <FiGithub /> GitHub
+              </ButtonLink>
+            </div>
+          </div>
+        </div>
+      </FeatureWrapper>
+      <Feature
+        title="GraphQL over HTTP compliant"
+        description={
+          <>
+            following the{' '}
+            <TextLink
+              href="https://graphql.github.io/graphql-over-http"
+              className="text-primary-500"
+            >
+              GraphQL over HTTP specification
+            </TextLink>
+          </>
+        }
+        gradient={0}
+      >
+        <div className="flex flex-col md:flex-row gap-12 justify-between">
+          <FeatureHighlights
+            highlights={[
+              {
+                title: 'Passes all audits',
+                description: (
+                  <>
+                    Tested using the graphql-http library.{' '}
+                    <TextLink
+                      href="https://github.com/enisdenjo/graphql-http/blob/master/implementations/graphql-yoga/README.md"
+                      className="text-primary-500"
+                    >
+                      See the report
+                    </TextLink>{' '}
+                    for more info.
+                  </>
+                ),
+                icon: (
+                  <Image
+                    src={httpImage}
+                    loading="eager"
+                    placeholder="empty"
+                    alt="Server over HTTP"
+                    className="w-full h-full"
+                  />
+                ),
+              },
+              {
+                title: 'Subscriptions',
+                description: (
+                  <>
+                    Built-in GraphQL Subscriptions over{' '}
+                    <TextLink
+                      href="https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events"
+                      className="text-primary-500"
+                    >
+                      Server-Sent Events
+                    </TextLink>
```

Comment:
I think we can link to https://www.the-guild.dev/graphql/yoga-server/docs/features/subscriptions instead.

The page then links to the MDN docs. IMHO there is no need to link of our page on the landing page